src/jobs/continue.yml

```diff
@@ -20,6 +20,12 @@ parameters:
     description: Path to the next config file to execute. By default, this will execute the "test-deploy" workflow.
     type: string
     default: .circleci/test-deploy.yml
+  circleci-api-host:
```

Comment:
Oops, @nise-nabe could we add the `type` of `string` to each of these parameters?

Reply:
fixed via 7fc6d14564ba2b3fa4327c3c3293b2c402224f8e